### ISSUE TITLE
Road to a stable 1.0?

### ISSUE THREAD

**Issue by mjbshaw** [alexcrichton/jobserver-rs#36]
I couldn't see a tracking issue or roadmap for a stable 1.0 release. Does one exist? If not, is that a desirable goal, and what might be needed to achieve it?

As a general rule of thumb I like to avoid unstable (0.*) crates, though jobserver has been stable enough for me that I've added to my list of exceptions.

(side note: thank you Alex! (and other contributors))

**Comment by alexcrichton** [comment]
No, I don't have any plans at this time. Not for any technical reason, though, just lack of effort.
